Ext4Pkg/Ext4Dxe/File.c

```diff
@@ -174,13 +176,13 @@ Ext4ReadSymlink (
   //
   // Allocate i_size_lo + 1

```

Comment:
This is wrong. why are you reading up to i_size_lo only?

Reply:
It is old commit